acme.sh

```diff
@@ -491,8 +491,14 @@ _post() {
     CURL="$CURL --dump-header $HTTP_HEADER "
     if [ "$needbase64" ] ; then
       response="$($CURL -A "User-Agent: $USER_AGENT" -X POST --data "$body" $url | _base64)"
+      if [ $? -gt 0 ] ;then
```

Comment:
Isn't that what I did in the last patch?

On Sat, May 7, 2016 at 1:25 AM, neil <notifications@github.com> wrote:

> In acme.sh
> <https://github.com/Neilpang/acme.sh/pull/158#discussion_r62413864>:
>
> > @@ -491,8 +491,14 @@ _post() {
> >      CURL="$CURL --dump-header $HTTP_HEADER "
> >      if [ "$needbase64" ] ; then
> >        response="$($CURL -A "User-Agent: $USER_AGENT" -X POST --data "$body" $url | _base64)"
> > +      if [ $? -gt 0 ] ;then
>
> so, why not add a variable "ret" here, and move this block to the end of
> this "_post()" function?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/Neilpang/acme.sh/pull/158/files/011d39ae23515ed1585053c68b9eefd629d97ae9#r62413864>
>